common/src/main/scala/com/gu/monitoring/SafeLogger.scala

```diff
@@ -0,0 +1,46 @@
+package com.gu.monitoring
```

Comment:
I actually just copy/pasted SafeLogger from membership-common but it's also used in support-frontend: https://github.com/guardian/support-frontend/blob/master/app/monitoring/Logging.scala 

Is there some sensible place for this to live where it can be used by both or shall I just keep this here?

Reply:
I think it's fine to leave it here. There are a couple of shared libraries eg. https://github.com/guardian/support-config & https://github.com/guardian/support-models but I don't think it particularly fits in either of those and it is a fairly small piece of code